src/components/views/messages/MessageActionBar.tsx

```diff
@@ -35,13 +36,17 @@ import Resend from "../../../Resend";
 import { MatrixClientPeg } from "../../../MatrixClientPeg";
 import { MediaEventHelper } from "../../../utils/MediaEventHelper";
 import DownloadActionButton from "./DownloadActionButton";
+import MessageContextMenu from "../context_menus/MessageContextMenu";
+import classNames from 'classnames';
+
 import SettingsStore from '../../../settings/SettingsStore';
 import { RoomPermalinkCreator } from '../../../utils/permalinks/Permalinks';
 import ReplyThread from '../elements/ReplyThread';
 
 interface IOptionsButtonProps {
     mxEvent: MatrixEvent;
-    getTile: () => any; // TODO: FIXME, haven't figured out what the return type is here
+    // TODO: Types
+    getTile: () => any | null;
```

Comment:
These types are still not consistent with what is actually used.

Reply:
@turt2live it can return `any` or `null`, `null` is a check for the future for `strictNullChecks` here and `any` is because if I do `unknown` here than tsc is extremely unhappy. We just need to fix this globally instead.

Reply:
The downstream types should be `any` then, not `unknown`. Consistency is important here for various non-public reasons.

Reply:
The proper one here would be `unknown`, but sure, fixed.